metrics/service.go

```diff
@@ -142,7 +143,7 @@ func (s *service) collect(ctx context.Context) {
 
 			logger.Infof("Status IngestionSamplesPerSecond=%0.2f SamplesIngested=%d IsHealthy=%v "+
 				"UploadQueueSize=%d TransferQueueSize=%d SegmentsTotal=%d SegmentsSize=%d UnhealthyReason=%s "+
-				"ActiveConnections=%d DroppedConnections=%d",
+				"ActiveConnections=%d DroppedConnections=%d MaxSegmentAgeSeconds=%f",
```

Comment:
Maybe %0.2f to keep format relatively consistent?

Reply:
thanks, addressed